src/main/java/org/opentest4j/MultipleFailuresError.java

```diff
@@ -34,15 +36,15 @@
 
 	private static final long serialVersionUID = 1L;
 
-	private final List<AssertionError> failures = new ArrayList<AssertionError>();
+	private final List<Throwable> failures = new ArrayList<Throwable>();
 
 	private final String heading;
 
 	public MultipleFailuresError(String heading) {
 		this.heading = isBlank(heading) ? "Multiple Failures" : heading.trim();
 	}
 
-	public void addFailure(AssertionError failure) {
+	public void addFailure(Throwable failure) {
```

Comment:
Unrelated to your change, but I cannot recall seeing a mutable exception type. Couldn't the list of throwables be passed in via the constructor? Or does Junit 5 not have that list at the time this exception is created?